2009-08-08  Yavor Doganov  <yavor@gnu.org>  (tiny change)

	* Documentation/gnustep-make.texi: Add @dircategory and @direntry
	commands.


--- gnustep-make-2.2.0.orig/Documentation/gnustep-make.texi
+++ gnustep-make-2.2.0/Documentation/gnustep-make.texi
@@ -8,6 +8,11 @@
 @smallbook
 @setchapternewpage odd
 
+@dircategory Software development
+@direntry
+* GNUstep Make: (gnustep-make).         The GNUstep build system.
+@end direntry
+
 @ifinfo
 
 Copyright @copyright{}  2000 Free Software Foundation, Inc.
